dynawo/sources/Common/DYNTraceStream.cpp

```diff
@@ -67,7 +67,7 @@ TraceStream::operator<<(const char* t) {
 void
 TraceStream::flush() {
   if (buffer_) {
-    Trace::log(slv_, tag_, buffer_->str());
+    Trace::instance().log(slv_, tag_, buffer_->str());
     buffer_->str(std::string());
```

Comment:
Trace::instance() missing

Reply:
wrong on my part

Reply:
buffer_ is part of TraceStream, not Trace. But instance shouldn't be called directly and should be used to delegate the static method to the instance method